staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -409,9 +419,15 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 		contextMetadata = metadata.New(map[string]string{"source": "cache"})
 	}
 
+	eventFreshDuration := config.EventsHistoryWindow
+	if eventFreshDuration < DefaultEventFreshDuration {
+		return nil, fmt.Errorf("cannot set eventFreshDuration to %v which is below %v", eventFreshDuration, DefaultEventFreshDuration)
```

Comment:
errors shouldn't reference local concepts. Better: `config.EventsHistoryWindow (%v) must not be below %v`

Reply:
Done.